src/styles/card-style.scss

```diff
@@ -0,0 +1,122 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.product {
+  justify-content: center;
+  flex-direction: column;
+  display: flex;
+  width: 198px;
```

Comment:
the product width should be 200px. 
Add `box-sizing: border-box` to `*` selector